docker-compose.yml

```diff
@@ -1,145 +0,0 @@
-# This is a Development Docker Composition of Specify 7
```

Comment:
This PR removes this file, so that should be undone :smile:

Reply:
Apologies on that - I wasn't sure how to handle since my local docker-compose had edits. I've reverted everything except for the change that fixes the issue. If I'm missing anything please let me know :sunglasses: